hieradata/dev/wso2/wso2is/pattern-4/key-manager.yaml

```diff
@@ -0,0 +1,210 @@
+
+wso2::template_list:
+  - repository/conf/datasources/bps-datasources.xml
+  - repository/conf/datasources/metrics-datasources.xml
+  - repository/conf/identity/identity.xml
+  - repository/conf/identity/sso-idp-config.xml
+  - repository/conf/identity/application-authentication.xml
+  - repository/conf/identity/EndpointConfig.properties
+  - repository/conf/datasources/am-datasources.xml
+#  - bin/ciphertool.sh
+
+# Provide the server list for setting up sso
+#wso2::sso_service_providers:
+#    wso2esb:
+#      assertion_consumer_service_url: https://esb.dev.wso2.org:9443/acs
+#      default_assertion_consumer_service_url: https://esb.dev.wso2.org:9443/acs
+#    wso2as:
+#      assertion_consumer_service_url: https://as.dev.wso2.org:9443/acs
+#      default_assertion_consumer_service_url: https://as.dev.wso2.org:9443/acs
+
+# wso2::ports:
+#   proxyPort:
+#     http: 80
+#     https: 443
+
+# File list to be copied to the product folder, if there are changes found in the files, the system
+# service will be restarted
+# wso2::file_list:
+#  - "repository/components/lib/%{hiera('wso2::datasources::mysql::connector_jar')}"
+#  - repository/conf/security/cipher-tool.properties
+#  - repository/conf/security/cipher-text.properties
+#  - repository/conf/security/secret-conf.properties
+
+# Files to be copied to the product folder when restarting the system service
+#wso2::service_refresh_file_list:
+#  - password-tmp
+
+wso2::clustering:
+  enabled: true
+  local_member_host: "%{::ipaddress}"
+  local_member_port: 4000
+  domain: is.wso2.domain
+# WKA membership scheme
+  membership_scheme: wka
+  wka:
+    members:
+      -
+        hostname: 192.168.100.111
+        port: 4000
+      -
+        hostname: 192.168.100.112
+        port: 4000
+# AWS membership scheme
+#   membership_scheme: aws
+#   aws:
+#     access_key: access-key
+#     secret_key: secret-key
+#     security_group: security-group 
+#     host_header: host-header
+#     region: region
+#     tag_key: tag-key
+#     tag_value: tag-value
+
+wso2::registry_mounts:
+  wso2_config_db:
+    path: /_system/config
+    target_path: /_system/config/is
+    read_only: false
+    registry_root: /
+    enable_cache: true
+
+  wso2_gov_db:
+    path: /_system/governance
+    target_path: /_system/governance
+    read_only: false
+    registry_root: /
+    enable_cache: true
+
+wso2::usermgt_datasource: wso2_user_db
+wso2::is_datasource: wso2_carbon_db
+
+# SSO Configuration
+wso2::sso_authentication:
+  enabled: false
+  login_page: /carbon/admin/login.jsp
+  service_provider_id: wso2is
+  sso_service_url: https://is.dev.wso2.org:9443/samlsso
+  consumer_service_url: https://is.dev.wso2.org:9443/acs
+
+wso2::master_datasources:
+ wso2_config_db:
+   name: WSO2_CONFIG_DB
+   description: The datasource used for config registry
+   driver_class_name: "%{hiera('wso2::datasources::mysql::driver_class_name')}"
+   url: jdbc:mysql://192.168.100.1:3306/WSO2_CONFIG_DB?autoReconnect=true
```

Comment:
Same here, shouldn't the URL be a generic one?

Reply:
Changed